native/src/main/java/io/ballerina/stdlib/graphql/runtime/schema/types/SchemaType.java

```diff
@@ -38,6 +38,7 @@
     private final Map<String, SchemaField> fields;
     private final List<Object> enumValues;
     private final Set<SchemaType> possibleTypes;
+    private final List<InputValue> inputFields;
```

Comment:
Why do we need `InputValue` list for a type? A type does not have arguments. Please read the spec!